dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -74,28 +72,21 @@ class EventOperationParamsMapper {
 
   private final CategoryOptionComboService categoryOptionComboService;
 
-  private final UserService userService;
-
   private final TrackedEntityAttributeService trackedEntityAttributeService;
 
   private final DataElementService dataElementService;
 
   private final OperationsParamsValidator paramsValidator;
 
   @Transactional(readOnly = true)
-  public EventQueryParams map(EventOperationParams operationParams)
+  public EventQueryParams map(EventOperationParams operationParams, UserDetails user)
```

Comment:
Can you add `@Nonnull` where applicable. Public methods should probably be enough to get static analysis help

Reply:
Done